libraries/rush-lib/src/cli/actions/UpdateAction.ts

```diff
@@ -63,6 +77,11 @@ export class UpdateAction extends BaseInstallAction {
   }
 
   protected async buildInstallOptionsAsync(): Promise<IInstallManagerOptions> {
+    const terminal: Terminal = new Terminal(new ConsoleTerminalProvider());
```

Comment:
Can we use `this._terminal` from the base class instead?

This terminal could conflict with that one. Also this terminal is not implementing the `--debug` flag.